src/org/usfirst/frc/team2399/robot/OI.java

```diff
@@ -129,5 +135,8 @@ public OI(){
 		climbStopButt.whenPressed(climbStopSpeed);
 		shiftToHotButt.whenPressed(shiftToHot);
 		shiftToDangerousButt.whenPressed(shiftToDangerous);
```

Comment:
This isn't related to this code review, but I'm getting on my soapbox here to ask "What does this even mean?"